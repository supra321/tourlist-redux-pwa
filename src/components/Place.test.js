import React from 'react';
import {shallow} from 'enzyme';
import Place from './Place';

it('Expect to render Place Component',()=>{
	expect(shallow(<Place />)).toMatchSnapshot();
})
