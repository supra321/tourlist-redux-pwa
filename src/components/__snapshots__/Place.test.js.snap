// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Expect to render Place Component 1`] = `ShallowWrapper {}`;
